src/pybind/mgr/dashboard/controllers/feedback.py

```diff
@@ -1,58 +1,111 @@
+        except RuntimeError as error:
+            if "Invalid issue tracker API key" in str(error):
+                raise DashboardException(msg='Error in creating tracker issue: Invalid API key',
+                                         component='feedback')
+            if "KeyError" in str(error):
+                raise DashboardException(msg=f'Error in creating tracker issue: {error}',
+                                         component='feedback')
+            raise DashboardException(msg=f'{error}',
+                                     http_status_code=500,
+                                     component='feedback')
+
+        return response
+
+
+@APIRouter('/feedback/api_key', Scope.CONFIG_OPT)
+@APIDoc("Feedback API", "Report")
+class FeedbackApiController(RESTController):
+
+    def list(self):
+        """
+        Returns Ceph tracker API key.
+        """
         try:
-            return self.tracker_client.create_issue(new_issue)
+            api_key = mgr.remote('feedback', 'get_api_key')
         except RequestException as error:
-            if error.status_code == 401:
-                raise DashboardException(msg=f'{"Invalid API key"}',
-                                         http_status_code=error.status_code,
-                                         component='feedback')
-            raise error
-        except Exception:
-            raise DashboardException(msg=f'{"API key not set"}',
-                                     http_status_code=401,
+            raise DashboardException(msg=f'{error}',
+                                     component='feedback')
+        if api_key is None:
+            raise DashboardException(msg='Issue tracker API key is not set',
+                                     http_status_code=400,
                                      component='feedback')
+        return api_key
 
-    def get(self, issue_number):
+    def create(self, api_key):
         """
-        Fetch issue details.
-        :param issueAPI: The issue tracker API access key.
+        Sets Ceph tracker API key.
+        :param api_key: The Ceph tracker API key.
         """
         try:
-            return self.tracker_client.get_issues(issue_number)
+            response = mgr.remote('feedback', 'set_api_key', api_key)
         except RequestException as error:
-            if error.status_code == 404:
-                raise DashboardException(msg=f'Issue {issue_number} not found',
-                                         http_status_code=error.status_code,
-                                         component='feedback')
-            raise error
+            raise DashboardException(msg=f'{error}',
+                                     component='feedback')
+        return response
+
+    def bulk_delete(self):
+        """
+        Deletes Ceph tracker API key.
+        """
+        try:
+            response = mgr.remote('feedback', 'delete_api_key')
+        except RequestException as error:
```

Comment:
AFAIK `mgr.remote` throws `RuntimeError` (or `ImportError`). Please see:
https://github.com/ceph/ceph/blob/master/src/pybind/mgr/mgr_module.py#L2106

```suggestion
        except RuntimeError as error:
```